lib/mapbox_gl.dart

```diff
@@ -23,3 +23,14 @@ part 'src/symbol.dart';
 part 'src/line.dart';
 part 'src/circle.dart';
 part 'src/ui.dart';
+
+final MethodChannel _staticChannel =
+MethodChannel('plugins.flutter.io/mapbox_gl');
+
+Future<void> installOfflineMapTiles(String tilesDb) async {
+  await _staticChannel.invokeMethod('installOfflineMapTiles',
+    <String, dynamic>{
+      'tilesdb': tilesDb,
+    },
+  );
+}
```

Comment:
can codeblock above be moved to a separate file? would prefer keeping the mapbox_gl.dart as clean as possible.

Reply:
done